boat_param:
    longitudinal_drag: 0.1
    lateral_drag: 0.1
    longitudinal_noise_var: 0.001
    lateral_noise_var: 0.001
    max_thrust: 2
    max_rudder_angle: 90      # in degree
    thrust_input_type: 0      # 0=step, 1=sin
    rudder_input_type: 0
sim_param:
    Ts: 0.01
meas_param:
    sigma_a: 0.001
    sigma_b: 0.001
    sigma_c: 0.001
    sigma_g: 0.001
    sigma_n: 0.001
    drift_stdev: 0.001
    pos_a: [10.0,10.0]
    pos_b: [-10.0, 10.0]
    pos_c: [10.0, -10.0]


